src/statics/Error403.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import { Link } from 'react-router';
+import steemconnect from 'steemconnect';
+import { FormattedMessage } from 'react-intl';
+import Header from '../app/Header';
+import Icon from '../widgets/Icon';
+
+const Error403 = () =>
+  <div className="main-panel">
+    <Header />
+    <div className="container my-5">
+      <div className="ptl text-center">
+        <h1><FormattedMessage id="page_forbidden" /></h1>
+        <h2><FormattedMessage id="page_forbidden_message" /></h2>
+        <p>You need to login to use the Chat feature, please
```

Comment:
We should remove this sentence. This component should not have some text specific to a page. Overwise we would not be able to use it on another page.